jekyll/_cci2/build-project.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: classic-docs
+title: "Build a Project"
+short-title: "Build a Project"
+description: "First project build on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
+This document describes how to configure your project to run on CircleCI 2.0. 
+
+## Prerequisites
+
+- Complete the steps in the [Join Beta & Sign Up]({{ site.baseurl }}/2.0/first-steps/) document. 
+- Install Docker, see [the Docker store](https://store.docker.com/search?offering=community&type=edition) for a free download  compatible with your operating system. 
+- Download a build image for your language, see [the CircleCI repo on hub.docker.com](https://hub.docker.com/u/circleci/).
+
+## Steps
+
+1. Create a directory called `.circleci` in the root directory of your local GitHub or Bitbucket code repository. 
+
+2. Create a `config.yml` file in the `.circleci` directory with the following lines, replacing *project root directory* with your project directory and *language:version* with your programming language and version number.
```

Comment:
May want to be more explicit that this is any docker image -- and the "language:version" is from the list available in the CircleCI Images list -- perhaps link to that page to help people see all the options?